app/controllers/employers/plan_years_controller.rb

```diff
@@ -39,7 +39,8 @@ def reference_plans
       Plan.by_active_year(params[:start_on]).for_service_areas(@employer_profile.service_area_ids).shop_market.health_coverage.by_carrier_profile(@carrier_profile).and(hios_id: /-01/)
     elsif params[:plan_option_kind] == "metal_level"
       @metal_level = params[:metal_level]
-      Plan.by_active_year(params[:start_on]).for_service_areas(@employer_profile.service_area_ids).shop_market.health_coverage.by_metal_level(@metal_level).and(hios_id: /-01/)
+      @available_carrier_ids = CarrierProfile.carriers_for(@employer_profile)
+      Plan.by_active_year(params[:start_on]).for_service_areas_and_carriers(@employer_profile.service_area_ids, @available_carrier_ids).shop_market.health_coverage.by_metal_level(@metal_level).and(hios_id: /-01/)
```

Comment:
Not sure this properly does the pairwise service_area_id/carrier profile id settings we talked about.